charts/timescaledb-single/values.yaml

```diff
@@ -297,6 +297,21 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+sharedMemory:
+  # By default Kubernetes only provides 64MB to /dev/shm
+  # /dev/shm is only used by PostgreSQL for work_mem for parallel workers,
+  # so most will not run into this issue.
+  # https://github.com/kubernetes/kubernetes/issues/28272
+  #
+  # If you do however run into:
+  #
+  #   SQLSTATE 53100
+  #   ERROR:  could not resize shared memory segment "/PostgreSQL.12345" to 4194304 bytes:
+  #   No space left on device
+  #
+  # you may wish to use a mount to Memory, by setting useMount to true
+  useMount: false
```

Comment:
I'm not sure what the best way is to expose this feature;
in a next iteration we'd probably work with feature flags; however in the current
file we mostly have:

```
feature:
  enabled: true
```

And I wanted to mimic that.